src/modules/I18n/i18next-scanner.config.js

```diff
@@ -4,13 +4,25 @@ const {
 const fg = require('fast-glob');
 const typescriptTransform = require('i18next-scanner-typescript');
 
-const namespaces = fg
+const moduleNamespaces = fg
   .sync(['src/modules/*', 'src/modulesAdmin/*', 'src/api/modules/*'], {
     onlyDirectories: true,
     deep: 1,
   })
   .filter((o) => !o.includes('__tests__'))
-  .map((o) => o.toLowerCase().replace('src/modules/', ''));
+  .map((o) => o.toLowerCase().replace('src/modules/', 'modules/'));
```

Comment:
I'm not sure to like this.

Translations of modules should be in given modules. Do you think it's feasible ?

Reply:
Yes i think it's feasible but i will make translations files in 2 separate path.